nanopub/client.py

```diff
@@ -1,54 +1,64 @@
-# -*- coding: utf-8 -*-
 """This module includes a client for the nanopub server.
 """
 
-import os
 import random
-import tempfile
 import warnings
-from typing import List, Union, Tuple
+from typing import List, Tuple, Union
 
 import rdflib
 import requests
 
-from nanopub import namespaces, profile
-from nanopub.definitions import DUMMY_NANOPUB_URI
-from nanopub.java_wrapper import JavaWrapper
-from nanopub.publication import Publication
-
-NANOPUB_GRLC_URLS = ["http://grlc.nanopubs.lod.labs.vu.nl/api/local/local/",
-                     "http://130.60.24.146:7881/api/local/local/",
-                     "https://openphacts.cs.man.ac.uk/nanopub/grlc/api/local/local/",
-                     "http://grlc.np.dumontierlab.com/api/local/local/"
-                     # These servers do currently not support
-                     # find_valid_signed_nanopubs_with_pattern (2020-12-21)
-                     # "https://grlc.nanopubs.knows.idlab.ugent.be/api/local/local/",
-                     # "http://grlc.np.scify.org/api/local/local/",
-                     ]
-NANOPUB_TEST_GRLC_URL = 'http://test-grlc.nanopubs.lod.labs.vu.nl/api/local/local/'
-NANOPUB_FETCH_FORMAT = 'trig'
-NANOPUB_TEST_URL = 'http://test-server.nanopubs.lod.labs.vu.nl/'
+from nanopub import namespaces
+from nanopub.definitions import DUMMY_NANOPUB_URI, NANOPUB_SERVER_LIST, NANOPUB_TEST_SERVER
+from nanopub.nanopub import Nanopub
+from nanopub.nanopub_conf import NanopubConf
+from nanopub.utils import log
+
+NANOPUB_GRLC_URLS = [
```

Comment:
I suppose we could put these URLs in some sort of config file? Or perhaps somewhere central like definitions.py? Not sure if that would really be better though (it's fine to leave it here for now)

Reply:
Yes currently `definitions.py` is what I used as central config file